app/server/controllers/UserController.js

```diff
@@ -117,10 +117,26 @@ UserController.loginWithPassword = function(email, password, callback){
   });
 };
 
+/**
+ * Creates and validates a new user given an email, password and profile.
+ * @param  {String}   email    User's email.
+ * @param  {String}   password User's password
+ * @param  {Object}   profile  Profile object
+ * @param  {Function} callback args(err, user)
+ */
+UserController.createValidUser = function(email, password, profile, callback){
+  User.validateProfile(profile, function(err){
```

Comment:
Where is this defined? Did this already exist? Or did you add this method for the User model object?

Reply:
validateProfile already exists in User.js